gridcapa-core-cc-post-processing-app/src/main/java/com/farao_community/farao/core_cc_post_processing/app/util/NamingRules.java

```diff
@@ -25,17 +25,20 @@ private NamingRules() {
-    public static final DateTimeFormatter UCT_OUTPUT_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-304_'yyyyMMdd'-F304-0V.zip'").withZone(IntervalUtil.ZONE_ID);
-    public static final DateTimeFormatter F305_FILENAME_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-305_'yyyyMMdd'-F305-0V.xml'").withZone(IntervalUtil.ZONE_ID);
-    public static final DateTimeFormatter CNE_OUTPUT_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-B06A43-299_'yyyyMMdd'-F299-0V.zip'").withZone(IntervalUtil.ZONE_ID);
-    public static final DateTimeFormatter OPTIMIZED_CB_FILENAME_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-B06A01-303_'yyyyMMdd'-F303-0V.xml'").withZone(IntervalUtil.ZONE_ID);
+    public static final DateTimeFormatter UCT_OUTPUT_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-304_'yyyyMMdd'-F304-<version>.zip'").withZone(IntervalUtil.ZONE_ID);
+    public static final DateTimeFormatter F305_FILENAME_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-305_'yyyyMMdd'-F305-<version>.xml'").withZone(IntervalUtil.ZONE_ID);
+    public static final DateTimeFormatter CNE_OUTPUT_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-B06A43-299_'yyyyMMdd'-F299-<version>.zip'").withZone(IntervalUtil.ZONE_ID);
+    public static final DateTimeFormatter OPTIMIZED_CB_FILENAME_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-B06A01-303_'yyyyMMdd'-F303-<version>.xml'").withZone(IntervalUtil.ZONE_ID);
     public static final String CGM_XML_HEADER_FILENAME = "CGM_XML_Header.xml";
-    public static final DateTimeFormatter LOGS_OUTPUT_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-342_'yyyyMMdd'-F342-0V.zip'").withZone(IntervalUtil.ZONE_ID);
-    public static final DateTimeFormatter METADATA_FILENAME_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-341_'yyyyMMdd'-F341-0V.csv'").withZone(IntervalUtil.ZONE_ID);
+    public static final DateTimeFormatter LOGS_OUTPUT_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-342_'yyyyMMdd'-F342-<version>.zip'").withZone(IntervalUtil.ZONE_ID);
+    public static final DateTimeFormatter METADATA_FILENAME_FORMATTER = DateTimeFormatter.ofPattern("'22XCORESO------S_10V1001C--00236Y_CORE-FB-341_'yyyyMMdd'-F341-<version>.csv'").withZone(IntervalUtil.ZONE_ID);
 
     public static String generateRF305FileName(LocalDate localDate) {
-        return NamingRules.F305_FILENAME_FORMATTER.format(localDate)
-            .replace("0V", String.format("%02d", 1));
+        return formatVersion(NamingRules.F305_FILENAME_FORMATTER.format(localDate), 1);
+    }
+
+    private static String formatVersion(String filename, int v) {
```

Comment:
Place this method either as the first or as the last method of your class, so as not to split the "generateXxx" methods in two separate groups.